# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../qmail-autoresponder/stricter_gcc.patch
# Copyright (C) 2006 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./main.c.orig	2004-05-06 19:29:15.000000000 -0400
+++ ./main.c	2004-05-06 19:30:23.000000000 -0400
@@ -244,18 +244,18 @@
 }
 
 static const char* usage_str =
-"usage: %s [-cqDNT] [-n NUM] [-s STR] [-t TIME] %s
-  -c       Copy message into response
-  -n NUM   Set the maximum number of replies (defaults to 1)
-  -s STR   Add the subject to the autoresponse, prefixed by STR
-  -t TIME  Set the time interval, in seconds (defaults to 1 hour)
-  -q       Don't show error messages
-  -D       Don't remove old response records
-  -N       Don't send, just send autoresponse to standard output
-  If more than NUM messages are received from the same sender
-  within TIME seconds of each other, no response is sent.
-  This program must be run by qmail.
-%s";
+"usage: %s [-cqDNT] [-n NUM] [-s STR] [-t TIME] %s\n"
+"  -c       Copy message into response\n"
+"  -n NUM   Set the maximum number of replies (defaults to 1)\n"
+"  -s STR   Add the subject to the autoresponse, prefixed by STR\n"
+"  -t TIME  Set the time interval, in seconds (defaults to 1 hour)\n"
+"  -q       Don't show error messages\n"
+"  -D       Don't remove old response records\n"
+"  -N       Don't send, just send autoresponse to standard output\n"
+"  If more than NUM messages are received from the same sender\n"
+"  within TIME seconds of each other, no response is sent.\n"
+"  This program must be run by qmail.\n"
+"%s";
 
 void usage(const char* msg)
 {
